tests/AuthenticationResultTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DotTest\Authentication;
+
+use Dot\Authentication\AuthenticationResult;
+use Dot\Authentication\Identity\IdentityInterface;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class AuthenticationResultTest extends TestCase
+{
+    protected AuthenticationResult $subject;
+    protected MockObject|IdentityInterface $indentyInterfaceMock;
+
+    protected function setUp(): void
+    {
+        $this->indentyInterfaceMock = $this->createMock(IdentityInterface::class);
+        $this->indentyInterfaceMock->method('getId')->will($this->returnValue(10));
+        $this->indentyInterfaceMock->method('getName')->willReturn('username');
+        $this->subject = new AuthenticationResult(2, 'valid', $this->indentyInterfaceMock);
+
+        parent::setUp();
```

Comment:
You can remove the call to `parent::setUp();`, because it does nothing.
The `setUp` method in `TestCase` is there only to make sure that the method is callable before each test, even if a test does not implement it.